### PR TITLE
Add inert global attribute

### LINKED ISSUE #19075: Add `inert` attribute
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes

### What specific section or headline is this issue about?

List of global attributes

### What information was incorrect, unhelpful, or incomplete?

The specifications list and the browser compat table already include the global `inert` attribute but the list of attributes is still missing it.

### What did you expect to see?

A description of the `inert` attribute.

### Do you have any supporting links, references, or citations?

There is already a separate page describing the related [`inert` DOM property](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/inert).

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/html/global_attributes`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/html/global_attributes/index.md
* Last commit: https://github.com/mdn/content/commit/5a9f418517a21e5f9d066cc507b2e4b28dbfbb95
* Document last modified: 2022-07-20T07:39:49.000Z

</details>

### PR DESCRIPTION
**Pull request by Isaac108**
Added the inert description. Along with a small page detailing its uses. More work is probably required for the page to be up to the elite standards as with the other attributes on the page, but I believe this is a good start.

#### Related issues

Fix #19075

#### Metadata

- [x] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by teoli2003** [comment]
Can you remove from this PR the unrelated pages? Thank you!

**Comment by Isaac108** [comment]
> 

Yep,, got it.

File: files/en-us/web/html/global_attributes/inert/index.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+The **`inert`** [global attribute](/en-US/docs/Web/HTML/Global_attributes)  in HTML can allows for the browser to disregard input events by the user.
+
+Specifically, it allows for many cool uses such as,
+
+•prevent the click event from generating
+
+•prevent implicit focus
+
+•prevent events from assistive technologies
```
Comment (teoli2003) [inline files/en-us/web/html/global_attributes/inert/index.md:27]:
```suggestion
Specifically, it prevents the following actions to happens:

- It prevents the {{domxref("Element/click_event", "click")}} event from being thrown when the user clicks on the element.
- It prevents the {{domxref("Element/focus_event", "focus")}} event to be raised, by preventing the focus to be given to the element.
- It prevents assistive technologies raised on this element.
```

**Review by teoli2003 — DISMISSED** [review]
Thank you!

In addition to these comments, we should have a small examples, as well as:
- a _Specifications_ section (using the `{{Specifications}}` macro
- a _Browser compatibility_ section (using the `{{Compat}}` macro
- a _See also_ section.

Is there already an entry in mdn/browser-compatibility-data ? If not, we need to add it there, and to add a YAML header here.

**Comment by teoli2003** [comment]
This will fixes #19075

**Comment by teoli2003** [comment]
Also, for the example, it should be possible to adapt the live example found [there](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/inert).

File: files/en-us/web/html/global_attributes/inert/index.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  - HTML
+  - Input
+  - Reference
+  - Text
+  - Web
+  - global
+  - inert
+browser-compat: html.global_attributes.inert
+---
+
```
[the remark this answers is not in the document]
Comment (Josh-Cena) [reply files/en-us/web/html/global_attributes/inert/index.md:16]: @teoli2003 note, after we start using Prettier, Prettier will enforce an empty line after the front matter. I suggest we start accepting empty lines here.

File: files/en-us/web/html/global_attributes/index.md [hunk 3]
```diff
@@ -72,6 +72,8 @@ In addition to the basic HTML global attributes, the following global attributes
   - : A Boolean attribute indicates that the element is not yet, or is no longer, _relevant_. For example, it can be used to hide elements of the page that can't be used until the login process has been completed. The browser won't render such elements. This attribute must not be used to hide content that could legitimately be shown.
 - [`id`](/en-US/docs/Web/HTML/Global_attributes/id)
   - : Defines a unique identifier (ID) which must be unique in the whole document. Its purpose is to identify the element when linking (using a fragment identifier), scripting, or styling (with CSS).
+- [`inert`](/en-US/docs/Web/HTML/Global_attributes/id)
+  - : A boolean value which makes the browser disregard user input events for the element. Useful where clicking events are present.
```
Comment (schalkneethling) [inline files/en-us/web/html/global_attributes/index.md:76]:
```suggestion
  - : A boolean value that makes the browser disregard user input events for the element. Useful when click events are present.
```

File: files/en-us/web/html/global_attributes/inert/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (teoli2003) [reply files/en-us/web/html/global_attributes/inert/index.md:16]: They are fixed by the DailyMarkdownLint scripts.

[the remark this answers is not in the document]
Comment (Josh-Cena) [reply files/en-us/web/html/global_attributes/inert/index.md:16]: When you search for `---\n\n` you'll see a lot of files with empty lines here—I think it's usually better to add an empty line.

File: files/en-us/web/html/global_attributes/index.md [hunk 4]
```diff
@@ -72,6 +72,8 @@ In addition to the basic HTML global attributes, the following global attributes
   - : A Boolean attribute indicates that the element is not yet, or is no longer, _relevant_. For example, it can be used to hide elements of the page that can't be used until the login process has been completed. The browser won't render such elements. This attribute must not be used to hide content that could legitimately be shown.
 - [`id`](/en-US/docs/Web/HTML/Global_attributes/id)
   - : Defines a unique identifier (ID) which must be unique in the whole document. Its purpose is to identify the element when linking (using a fragment identifier), scripting, or styling (with CSS).
+- [`inert`](/en-US/docs/Web/HTML/Global_attributes/id)
+  - : A boolean value that makes the browser disregard user input events for the element. Useful when click events are present.
```
Comment (bkardell) [inline files/en-us/web/html/global_attributes/index.md:76]: I realize these are very brief, but maybe this is too brief?  What about like "A boolean value that indicates that the element (and all of its flat tree descendants) are not currently relevant and should act as if it were not present for purposes of user interaction (including hiding it from accessibility technology)"

File: files/en-us/web/html/global_attributes/inert/index.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+{{HTMLSidebar("Global_attributes")}}
+
+The **`inert`** [global attribute](/en-US/docs/Web/HTML/Global_attributes) instructs the browser to disregard input events sent by the user, including focus-related events, as well as events generated by assistive technologies.
+
+Specifically, it prevents the following actions to happens:
+
+- It prevents the {{domxref("Element/click_event", "click")}} event from being thrown when the user clicks on the element.
+- It prevents the {{domxref("Element/focus_event", "focus")}} event to be raised, by preventing the focus to be given to the element.
+- It prevents assistive technologies raised on this element.
```
Comment (bkardell) [inline files/en-us/web/html/global_attributes/inert/index.md:25]: Not sure what this means? It is hidden from the a11y tree

File: files/en-us/web/html/global_attributes/index.md [hunk 6]
```diff
@@ -72,6 +72,8 @@ In addition to the basic HTML global attributes, the following global attributes
   - : A Boolean attribute indicates that the element is not yet, or is no longer, _relevant_. For example, it can be used to hide elements of the page that can't be used until the login process has been completed. The browser won't render such elements. This attribute must not be used to hide content that could legitimately be shown.
 - [`id`](/en-US/docs/Web/HTML/Global_attributes/id)
   - : Defines a unique identifier (ID) which must be unique in the whole document. Its purpose is to identify the element when linking (using a fragment identifier), scripting, or styling (with CSS).
+- [`inert`](/en-US/docs/Web/HTML/Global_attributes/id)
```
Comment (caugner) [inline files/en-us/web/html/global_attributes/index.md:75]: Shouldn't it link to `Global_attributes/inert`?

```suggestion
- [`inert`](/en-US/docs/Web/HTML/Global_attributes/inert)
```

**Review by caugner — DISMISSED** [review]

File: files/en-us/web/html/global_attributes/inert/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (teoli2003) [reply files/en-us/web/html/global_attributes/inert/index.md:16]: I'm fine this, then.

File: files/en-us/web/html/global_attributes/inert/index.md (hunk 5 above)
Comment (teoli2003) [reply files/en-us/web/html/global_attributes/inert/index.md:25]: What about:
```suggestion
- It hides the element and its content from assistive technologies by excluding them from the accessibility tree.
```

File: files/en-us/web/html/global_attributes/index.md (hunk 6 above)
Comment (teoli2003) [reply files/en-us/web/html/global_attributes/index.md:75]: Yes, of course.

File: files/en-us/web/html/global_attributes/inert/index.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
+---
+title: inert
+slug: Web/HTML/Global_attributes/inputmode
```
Comment (teoli2003) [inline files/en-us/web/html/global_attributes/inert/index.md:3]:
```suggestion
slug: Web/HTML/Global_attributes/inert
```

File: files/en-us/web/html/global_attributes/inert/index.md (hunk 5 above)
Comment (teoli2003) [reply files/en-us/web/html/global_attributes/inert/index.md:25]: I went ahead and committed this, but we can still work on line 25, if you want.

File: files/en-us/web/html/global_attributes/inert/index.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+{{HTMLSidebar("Global_attributes")}}
+
+The **`inert`** [global attribute](/en-US/docs/Web/HTML/Global_attributes) instructs the browser to disregard input events sent by the user, including focus-related events, as well as events generated by assistive technologies.
+
+Specifically, it prevents the following actions to happens:
+
+- It prevents the {{domxref("Element/click_event", "click")}} event from being thrown when the user clicks on the element.
+- It prevents the {{domxref("Element/focus_event", "focus")}} event to be raised, by preventing the focus to be given to the element.
+- It hides the element and its content from assistive technologies by excluding them from the accessibility tree.
```
Comment (Josh-Cena) [inline files/en-us/web/html/global_attributes/inert/index.md:25]:
```suggestion
Specifically, it does the following:

- It prevents the {{domxref("Element/click_event", "click")}} event from being thrown when the user clicks on the element.
- It prevents the {{domxref("Element/focus_event", "focus")}} event from being raised, by preventing the element from gaining focus.
- It hides the element and its content from assistive technologies by excluding them from the accessibility tree.
```

**Comment by Josh-Cena** [comment]
Seems we have BCD. Is this ready?

**Comment by sideshowbarker** [comment]
> Seems we have BCD. Is this ready?

yeah, seems like it’s ready to go — though at https://github.com/mdn/content/pull/19090#discussion_r991645879 I see one pending commit suggestion

**Review by caugner — COMMENTED** [review]
No objections from me, but I'll leave the content review to others.

**Comment by queengooborg** [comment]
Hey @teoli2003, are you still planning on reviewing this PR?

**Review by estelle — APPROVED** [review]

**Comment by estelle** [comment]
Congratulations on your first MDN merged PR. Thank you for your contribution.

**Pull request merged**